### PR TITLE
adding logging to services and update read me

### PR DESCRIPTION
**Pull request by jules203**
# Pull Request Template

## Checklist:

- [ ] I have written test cases for this change, and my code passed the test - **needed for this?**
- [x] I have updated documentation to reflect this change
- [x] I have attached this PR to an issue/user story
 I canceled the old PR for logging and made this new one

### CODE REVIEW

**Comment by KHafner** [comment]
Should config.py be included as part of this PR? If that is included the checks should pass, correct?

**Comment by jules203** [comment]
@KHafner Added the config.py file!

**Comment by iLtc** [comment]
Please run `git pull` and `git merge master` to merge all the latest changes to your branch. This should allow all the tests to pass.

File: service/models.py [hunk 1]
```diff
@@ -166,16 +166,92 @@ def deserialize(self, data: dict):
             )
         return self
 
+ @classmethod
```
Comment (iLtc) [inline service/models.py:169]: Please fix this indent issue.

**Review by iLtc — CHANGES_REQUESTED** [review]

**Review by iLtc — APPROVED** [review]
LGTM, thanks!

**Review by KHafner — APPROVED** [review]

**Pull request merged**